availability_zones.yaml

```diff
@@ -1,22 +1,24 @@
-
-online:
-  - online
-EU:
-  - online
-  - EU
-NERSC:
-  - online
-  - NERSC
-NCAR:
-  - online
-  - NCAR
-UK:
-  - online
-  - UK
-JAPAN:
-  - online
-  - JAPAN
-
-CN:
-  - online
-  - CN
\ No newline at end of file
+
+online:
+  - online
+EU:
+  - online
+  - EU
+NERSC:
+  - online
+  - NERSC
+NCAR:
+  - online
+  - NCAR
+UK:
+  - online
+  - UK
+JAPAN:
+  - online
+  - JAPAN
+
+CN:
+  - online
+  - CN

```

Comment:
This commit changes all of the lines in the current file.
I would have expected to see a change which only adds an entry for `SP`, but does't modify the others.